driver/optmaindriver.ml

```diff
@@ -161,5 +161,5 @@ let main unix argv ppf ~flambda2 =
     Location.report_exception ppf x;
     2
   | () ->
-    Profile.print Format.std_formatter !Clflags.profile_columns;
+    Profile.print Format.std_formatter !Clflags.profile_columns ~timings_precision:!Clflags.timings_precision;
```

Comment:
Please add the new flag to OCAMLPARAM, it'd be in `ocaml/driver/compenv.ml` near the case for "timings" and "profile", and look something like this:
```
| "timings-precision" ->
  int_setter ppf name  timings_precision v 
```

Reply:
Added.